javascript/javascript1/week4/sentimentAnalyzer.js

```diff
@@ -0,0 +1,36 @@
+const allowedPositiveWords=["super", "awesome", "happy", "excited"];
+const allowedNegativeWords=["hate", "boring", "sad", "unhappy"];
+
+function getSentimentScore(incomingString)
+{
+    let result=
+    {
+        scrore:0,
+        positiveWords:[],
+        negativeWords:[]
+    }
+    const wordString=incomingString.split('');
+    for(let i=0;i<wordString.length;i++)
+    {
+        const myWord=wordString[i];
+        if(allowedPositiveWords.includes(myWord))
+        {
+            result.score++;
+            result.positiveWords.push(myWord);
+
+        }
+        else if(allowedNegativeWords.includes(myWord))
+        {
+            result.score--;
+            result.negativeWords.push(myWord);
+        }
+        
+    }
+    return result;
+}
+
+const positiveSentence=getSentimentScore("I am mega super awesome happy");
+console.log(positiveSentence);
```

Comment:
you can make your console.logs nicer. Put some words on it, right now it will just output the whole object without context. If you have a big program, you won't know what this output means.